### PR TITLE
Issue 51 sync with backend axios config

### PR DESCRIPTION
**Pull request by DelFlumen**
develop

## GitHub Board

**Issue link**

[#51 ]( )

**Story link**

## Code reviewers
- [ ] @github_username

## Summary of issue

- [ ]  Summary of issue
- [ ]  Summary of issue

## Summary of change

- [ ]  Summary of change
- [ ]  Summary of change

### CODE REVIEW

File: src/lib/utilities/API/types.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      mainInstitution: {
+        city: {
+          id: number;
+          name: string;
+        };
+        id: number;
+        name: string;
+      };
+      qualification: string;
+    },
```
Comment (SergeOlabin) [inline src/lib/utilities/API/types.ts:73]: this should be a separate interface

File: src/lib/utilities/API/api.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    return instance.get<GetResponseType<ImportantPostResponseType>>(
+      `/post/${postsType}`,
+      config,
+    );
+  }
+  return instance.get<GetResponseType<LatestPostResponseType>>(
+    `/post/${postsType}`,
+    config,
+  );
+};
```
Comment (SergeOlabin) [inline src/lib/utilities/API/api.ts:55]: pls make 2 methods:
`getImportantPosts()`
`getLatestPosts()`

**Review by SergeOlabin — CHANGES_REQUESTED** [review]
pls merge develop to your branch so that action will be fired on PR to develop

File: src/lib/utilities/API/types.ts (hunk 1 above)
Comment (DelFlumen) [reply src/lib/utilities/API/types.ts]: fixed

**Review by SergeOlabin — COMMENTED** [review]
pls check sonar Code Smells section - it should be fixed.
https://sonarcloud.io/project/issues?id=ita-social-projects_dokazovi-fe&pullRequest=82&resolved=false&types=CODE_SMELL

**Pull request merged**